R/biokinetics.R

```diff
@@ -534,7 +552,10 @@ biokinetics <- R6::R6Class(
           by = c("calendar_day", "titre_type"))
       }
 
-      dt_out[, time_shift := time_shift]
+      dt_out <- dt_out[, time_shift := time_shift]
+      class(dt_out) <- append("biokinetics_individual_trajectories", class(dt_out))
```

Comment:
I haven't added a plot function for the individual trajectories yet, since this PR is already quite big. But will use the same approach - making the individual trajectories an S3 class and defining a default plot function for objects of this class.

Reply:
yeah that seems sensible, good to keep a consistent workflow, lots of other uses for the individual-level trajectories other than plotting too!